staging/src/k8s.io/apiserver/pkg/storage/cacher.go

```diff
@@ -62,8 +62,8 @@ type CacherConfig struct {
 	// KeyFunc is used to get a key in the underlying storage for a given object.
 	KeyFunc func(runtime.Object) (string, error)
 
-	// GetAttrsFunc is used to get object labels and fields.
-	GetAttrsFunc func(runtime.Object) (labels.Set, fields.Set, error)
+	// GetAttrsFunc is used to get object labels, fields, and the uninitialized bool
+	GetAttrsFunc func(runtime.Object) (labels.Set, fields.Set, bool, error)
```

Comment:
consider naming the parameters for clarity (does true = initialized or uninitialized?)